lib/base/networking/api/gocast/api_v2.pbenum.dart

```diff
@@ -0,0 +1,37 @@
+//
+//  Generated code. Do not modify.
```

Comment:
It depends on whether you want to have to re-build the files when cloning the repository or not. I would suggest leaving it in as otherwise you would always have to run `dart pub global activate protoc_plugin
export PATH="$PATH:$HOME/.pub-cache/bin"
curl -o proto/gocast/api_v2.proto https://raw.githubusercontent.com/TUM-Dev/gocast/IPraktikum-36-user-endpoints/api_v2/api_v2.proto
protoc --dart_out=grpc:lib/base/networking/api -I./proto google/protobuf/timestamp.proto google/protobuf/empty.proto proto/gocast/api_v2.proto` which could confuse some.

Reply:
We can have a shell script maybe ?

Reply:
```bash
#!/bin/bash

# Stop the script if any command fails
set -e

# Activate the protoc_plugin globally
dart pub global activate protoc_plugin

# Add Dart's package cache bin directory to the PATH
export PATH="$PATH:$HOME/.pub-cache/bin"

# Create a directory for the proto files if it doesn't exist
mkdir -p proto/gocast

# Download the api_v2.proto file
curl -o proto/gocast/api_v2.proto https://raw.githubusercontent.com/TUM-Dev/gocast/IPraktikum-36-user-endpoints/api_v2/api_v2.proto

# Run the protoc command to generate Dart gRPC code
protoc --dart_out=grpc:lib/base/networking/api -I./proto google/protobuf/timestamp.proto google/protobuf/empty.proto proto/gocast/api_v2.proto

echo "gRPC code generation completed successfully."

```

Reply:
Yes, that looks good. I'll add this `./setup_grpc.sh` so that one can easily update the generated proto files (if the API has been updated), but I'll leave the automatically generated files in the repo so that it's less confusing for others who start working on the client.